### DISCUSSION TITLE
Welcome to cityjson2jsonfg Discussions!

### DISCUSSION THREAD

**Discussion by balazsdukai** [3DGI/cityjson2jsonfg#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by StefanCiesilski** [comment]
Hi @balazsdukai ,

Thanks for the tool.

Is support for CityJSON: 2.0.* and JSON-FG: 0.3.0 planned?

I checked the converted file with an ogc-checker (https://geonovum.github.io/ogc-checker/#/json-fg). It notes that

"conformsTo": ["[ogc-json-fg-1-0.2:core]", "[ogc-json-fg-1-0.2:3d]"], is not in "uri" format. 
--> e.g., http://www.opengis.net/spec/json-fg-1/0.3/conf/core …

All the data is written to a single line. Is a pretty print possible?

Thanks, Stefan

**Reply by balazsdukai** [reply]
Hi @StefanCiesilski , true, it doesn't pretty print at the moment. It is intentional, because spaces and newlines in a file take a lot of...space 😄 . But it would be a small change to add it, wired into a cli flag. I would be happy to review the PR if you would open one.

I haven't really kept up with the changes in JSON-FG, so I need to check what would it mean to support 0.3.0

CityJSON v2.0 support over v1.1 is mostly a matter of updating the cjio dependency.

**Reply by StefanCiesilski** [reply]
Hi @balazsdukai ,
you're right. A CLI flag would definitely the best. Unfortunately, I can't help you with a PR.

I can't tell you exactly what changed in version 0.3.0. I only saw that there's a new version.

Thanks for your quick reply.
